### PR TITLE
Tooltip fixes

### PR DESCRIPTION
**Pull request by xavijam**
- Using a wrapper class instead of the proper CDB-Tooltip class, as we do for new infowindow.
- Fixed CSS classes.
- Added the tooltip and other new components to the v4-layergroup example.

CR: @alonsogarciapablo

### CODE REVIEW

**Comment by xavijam** [comment]
retest this please

File: src/geo/ui/tooltip.js [hunk 1]
```diff
@@ -148,10 +148,15 @@ var Tooltip = InfoBox.extend({
     var mapViewSize = this.options.mapView.getSize();
     var top = 0;
     var left = 0;
-    var modifierClass = 'CDB-Tooltip--';
+    var modifierClass = 'CDB-Tooltip-wrapper--';
```
Comment (alonsogarciapablo) [inline src/geo/ui/tooltip.js]: We can get this class name from `this.className`. Here's an idea:

```
setPosition: function () {
  // ...
 this._removePositionModifiers()
  // ...
},

_removePositionModifiers: function ( ) {
  var positions = [ 'topLeft', 'topRight', 'bottomRight', 'bottomLeft' ];
  var positionModifiers = _.map(position, function (className) {
    return this._modifierClassName(className);
  }, this);
  this.$el.removeClass(positionModifiers);
},

_modifierClassName: function (className) {
  return this.className + '--' + className;
})

```

**Comment by alonsogarciapablo** [comment]
retest this please

**Comment by xavijam** [comment]
retest this please

**Pull request merged**